doc/misc.rst

```diff
@@ -16,7 +16,16 @@ This set of instructions is intended for 64-bit Linux and macOS computers.
     Everywhere else, just making sure you have the ``g++`` package should be
     enough.
 
-#.  Install your favorite variant of `miniforge <https://github.com/conda-forge/miniforge>`_.
+#.  Install your favorite variant of `miniforge <https://github.com/conda-forge/miniforge>`_:
+
+        # pick one
+        curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh
+        curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh
+        curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh
```

Comment:
After https://github.com/conda-forge/miniforge/pull/156,
```
curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh
```

Reply:
Thanks! e147d8b. Also I wish I hadn't just https://github.com/inducer/grudge/pull/94 and https://github.com/inducer/meshmode/pull/190, but :shrug: such is life. :)